generate-xml/domain.nix

```diff
@@ -39,6 +39,10 @@ let
             (subelem "node" [ (subattr "id" typeInt) (subattr "bandwidth" typeInt) ] [ ])
           ])
         ])
+        (subelem "numatune" [ ] [
+          (subelem "memory" [ (subattr "mode" typeString) (subattr "nodeset" typeString) ] [ ])
+          (subelem "memnode" [ (subattr "mode" typeString) (subattr "nodeset" typeString) (subattr "cellid" typeString) ]  [ ])
+        ])
```

Comment:
Should `numatune.memnode.cellid` be `typeInt` instead?

Reply:
yes, I think so - I can't think of a case where cellid would not be an integer